modules/ebi-metagenomics/combinedgenecaller/merge/main.nf

```diff
@@ -7,7 +7,7 @@ process COMBINEDGENECALLER_MERGE {
         'biocontainers/mgnify-pipelines-toolkit:0.2.0--pyhdfd78af_0' }"
```

Comment:
just FYI, the most recent version of the toolkit is 0.2.1 now

Reply:
ahh .. perfect.